src/store/reducers/tezos/transactions.ts

```diff
@@ -15,13 +15,4 @@ const transactions = (state = initState, action: SetTransactionsAction): Transac
   }
 };
 
-const errors = (state = initState, action: any): any => {
```

Comment:
Dlaczego to wyrzucamy? Nie przyda się już? Może zakomentować?

Reply:
ok zakomentowałem na wszelki wypadek